test/e2e/scenario/rum/errors.scenario.ts

```diff
@@ -1,20 +1,159 @@
-import { createTest } from '../../lib/framework'
-import { browserExecute, withBrowserLogs } from '../../lib/helpers/browser'
+import { RumErrorEvent } from '@datadog/browser-rum-core'
+import { createTest, html } from '../../lib/framework'
+import { withBrowserLogs } from '../../lib/helpers/browser'
 import { flushEvents } from '../../lib/helpers/flushEvents'
 
+// Note: using `browserExecute` to throw exceptions may result in "Script error." being reported,
+// because WDIO is evaluating the script in a different context than the page.
+function createBody(errorGenerator: string) {
+  return html`
+    <button>click me</button>
+    <script>
+      const btn = document.querySelector('button')
+      btn.addEventListener('click', function handler() {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** avoid abbreviation‏
```suggestion
      const button = document.querySelector('button')
      button.addEventListener('click', function handler() {
```